docs/src/main/asciidoc/application-configuration-guide.adoc

```diff
@@ -360,6 +360,48 @@ You can override these runtime properties with the following mechanisms (in decr
 
 NOTE: Environment variables names are following the conversion rules of link:https://github.com/eclipse/microprofile-config/blob/master/spec/src/main/asciidoc/configsources.asciidoc#default-configsources[Eclipse MicroProfile]
 
+== Using local configuration file during Development
+As an alternative to defining some configuration variables via host environment variables during development, you can define configuration variables in a local configuration file via a `quarkus.local-config` property.
+This is practical as it allows each member of your development team to customize and manage the development environment via a file.
+Doing so, Quarkus will load configuration variables having higher priority than variable defined in a `application.properties` file but with a lower priority than host environment variables.
+
+See the example below.
+
+Let's create a `src/main/resources/.local-config.properties` file and add the following variable.
```

Comment:
Isn't the point of a local config that it would be stored in your home directory, e.g. `${user.home}/.application.properties`?  That was my understanding anyway, maybe I missed something though.

Is there a use case for putting it within the project?

Reply:
The `quarkus.local-config` permits us to point to any file within the file system. 
It is also easier to manage config file (local or not) that resides within the project rather than outside hence the choice.

Reply:
Nevertheless, `src/main/resources` is generally not on the file system.  The files within the project are generally included as resources.  So is the local config a resource or a file?  If it's a file then putting it in the resources directory is confusing if not outright wrong.

Reply:
> If it's a file then putting it in the resources directory is confusing if not outright wrong.

You are right and I have adjusted the example taking into account this feedback.